templates/components/sections_datatable.html.twig

```diff
@@ -0,0 +1,131 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2025 Teclib' and contributors.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% extends 'components/datatable.html.twig' %}
+{% import 'components/form/fields_macros.html.twig' as macro %}
+
+{% block datatable_entries %}
+    {% for step in validationsteps %}
+        {% if step.entries|length > 0 %}
+
+            {# edit button #}
+            {% set edit_step_button %}
+                {{ step.edit_link_js.js|raw }}
+                <a href=""
+                   role="button"
+                   title="{{ __('Edit validation step') }}"
+                   data-bs-toggle='modal'
+                   data-bs-target="#{{ step.edit_link_js.target }}"
+                   {{ step.edit_link_js.onclick|raw }}
+                >
+                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none"
```

Comment:
For tabler icons, we use the font format in 99% of the case: `<i class="ti ti-xxx"></i>`